package.json

```diff
@@ -6,7 +6,8 @@
   "scripts": {
     "start": "node .",
     "stop": "pkill levweb",
-    "dev": "NODE_ENV=development nodemon .",
+    "dev": "npm run mock-kc-proxy & NODE_ENV=development nodemon .",
```

Comment:
I thought that & pushed the process to the background where it won't be killed. At least the old api mock was not killed in this way which is why we had the explicit stop:mockapi script.